src/config/dark-sites.config

```diff
@@ -544,6 +551,7 @@ protogen.dev
 protondb.com
 proxx.app
 psi.cynicaloptimist.me
+ptable.com
```

Comment:
Please remove this entry.
![image](https://user-images.githubusercontent.com/25481501/110013138-2eeefa00-7d19-11eb-80aa-31f215cb64b4.png)